backend/src/nodes/onnx_nodes.py

```diff
@@ -180,3 +183,31 @@ def run(
             in_nc,
             lambda i: self.upscale(i, session, split_factor, change_shape),
         )
+
+
+@NodeFactory.register("chainner:onnx:convert_to_ncnn")
+class ConvertOnnxToNcnnNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = """Convert an ONNX model to NCNN."""
+        self.inputs = [OnnxModelInput("ONNX Model"), OnnxFpDropdown()]
+        self.outputs = [NcnnModelOutput("NCNN Model"), TextOutput("FP Mode")]
```

Comment:
Could you type the FP mode output? This should be fairly simple and would be a nice introduction into working with the type system.

This will be pretty much analogous to the [`StackOrientationDropdown`](https://github.com/joeyballentine/chaiNNer/blob/3d57f4ddd379b1df32f04c1f3573cf0827ef0a7e/backend/src/nodes/properties/inputs/generic_inputs.py#L133). You need to [define](https://github.com/joeyballentine/chaiNNer/blob/3d57f4ddd379b1df32f04c1f3573cf0827ef0a7e/src/common/types/chainner-scope.ts#L65) an [enum](https://github.com/joeyballentine/chaiNNer/blob/main/src/common/types/navi.md#enum-definitions) for the Fp dropdown and then [match](https://github.com/joeyballentine/chaiNNer/blob/main/src/common/types/navi.md#match) the dropdown variants in the text output.